scripts/lrw.sh

```diff
@@ -360,10 +365,15 @@ setup_environment() {
   sh -c "cd liquidity && pipenv install --dev"
 
   info "***Setup Liquidity Provider***"
-  (cd liquidity; pipenv run python setup_env.py || exit 1) || fail "Liquidity service setup failed"
+  (cd liquidity; BLOCKCHAIN=${chain_type} pipenv run python setup_env.py || exit 1) || fail "Liquidity service setup failed"
 
-  info "***Setting up environment .env files***"
-  PIPENV_PIPFILE=backend/Pipfile pipenv run python3 scripts/set_env.py || fail "Wallet setup failed"
```

Comment:
should use one set_env script, do not create a specific set env for premainnet.
you should generate what need to be generated, and pass the rest as env variables, which is set by the deploy script, instead of creating a new set_env_premainnet script to set those static values.